crates/bevy_reflect/src/path/access.rs

```diff
@@ -125,9 +161,14 @@ impl fmt::Display for Access<'_> {
 }
 
 impl<'a> Access<'a> {
-    pub(super) fn into_owned(self) -> Access<'static> {
+    /// If the [`Access`] is of variant [`Field`](Access::Field),
+    /// the field's [`Cow<str>`] will be converted to it's owned
+    /// counterpart, which doesn't require a lifetime.
+    ///
+    /// Otherwise does nothing.
+    pub fn as_static(self) -> Access<'static> {
```

Comment:
Why the rename? I think `to_owned` is a bit clearer (i.e. this doesn't create a static reference) and more consistent with other Bevy APIs.

Reply:
I disagree. Calling the method `to_owned` makes implies it takes in a reference and makes an owned version of it. `as_static` makes it very clear what the method does. `Access<'a>` goes in, `Access<'static>` goes out. No need to know how `Cow` works to know what this method does. (It actually confused me when I first looked at it)

Reply:
Sorry I meant to write `into_owned`. This is what Bevy conventionally names methods that turn an `'a` into a "`'static`" (quotes around it because imo it's not truly static, it's just owned data). For example, [`AssetPath::into_owned`](https://github.com/bevyengine/bevy/blob/dc8fc6cb34d6064530c698f30445cd69d1c79682/crates/bevy_asset/src/path.rs#L284) does something similar. I don't think we use the phrasing of `as_static` for any public API (currently anyways).

Reply:
Ah, alright. Fair enough.